funnel/forms/venue.py

```diff
@@ -27,10 +27,14 @@ class VenueForm(forms.Form):
         validators=[forms.validators.Optional(), forms.validators.Length(max=20)])
     country = forms.SelectField(__("Country"),
         validators=[forms.validators.Optional(), forms.validators.Length(max=2)],
-        choices=country_codes, default="IN")
+        choices=[], default='IN')
     coordinates = forms.CoordinatesField(__("Location"), description=__("Pick a location on the map"),
         validators=[forms.validators.Optional(), forms.validators.ValidCoordinates()])
 
+    def set_queries(self):
+        pycountry_locale = gettext.translation('iso3166-2', pycountry.LOCALES_DIR, languages=[get_locale()])
+        self.country.choices = [(country.alpha_2, pycountry_locale.gettext(country.name)) for country in pycountry.countries]
```

Comment:
This won't be sorted if we're doing localization outside the source list. We need to sort it again.